provider/infoblox/infoblox.go

```diff
@@ -166,40 +183,23 @@ func (p *InfobloxProvider) Records(ctx context.Context) (endpoints []*endpoint.E
 	}
 
 	for _, zone := range zones {
-		logrus.Debugf("fetch records from zone '%s'", zone.Fqdn)
+		endpointsTypeA := make([]*endpoint.Endpoint, 0)
 		var resA []ibclient.RecordA
+		logrus.Debugf("fetch records from zone '%s'", zone.Fqdn)
 		objA := ibclient.NewRecordA(
 			ibclient.RecordA{
 				Zone: zone.Fqdn,
 				View: p.view,
 			},
 		)
+
 		err = p.client.GetObject(objA, "", &resA)
 		if err != nil {
 			return nil, fmt.Errorf("could not fetch A records from zone '%s': %s", zone.Fqdn, err)
 		}
 		for _, res := range resA {
-			newEndpoint := endpoint.NewEndpoint(res.Name, endpoint.RecordTypeA, res.Ipv4Addr)
-			if p.createPTR {
-				newEndpoint.WithProviderSpecific(providerSpecificInfobloxPtrRecord, "false")
-			}
```

Comment:
Same question for this part: Why is it removed and how is this addressed now?

Reply:
Hi @BobVanB like 4 the H records this is addressed at line 311 (end of the function) for both A and H records.

Reply:
Yes i figured it would be the same.
That was last part i was looking at. Still trying to make the change smaller, but i afraid the multitarget one is not going to be smaller.
Thanks for your reply.